apps/web-giddh/src/app/invoice/templates/edit-template/filters-container/content-filters/content.filters.component.html

```diff
@@ -25,14 +25,12 @@ <h2>Header</h2>
                 <div *ngIf="fieldsAndVisibility.header.gstin" class="form-group">
                     <input type="checkbox" [(ngModel)]="customTemplate.sections['header'].data['gstin'].display" (ngModelChange)="onChangeFieldVisibility(null, null, null)" checked /> GSTIN
                 </div>
-                <ng-template>
-                    <div class="form-group clearfix customField">
-                        <input checked type="checkbox">
-                        <span>
-                            <input class="form-control" type="text" value="Registered under Composition Scheme">
-                        </span>
-                    </div>
-                </ng-template>
+                <div class="form-group clearfix customField" *ngIf="(customTemplate?.templateType === 'gst_template_a' || customTemplate?.templateType === 'gst_template_e') && voucherType==='sales'">
+                    <input type="checkbox" name="gstCompositionLabel" [(ngModel)]="customTemplate.sections['header'].data['gstComposition'].display" (ngModelChange)="onChangeFieldVisibility(null, null, null)" checked />
+                    <span>
+                        <input class="form-control" [maxlength]="70" type="text" name="gstComposition" placeholder="Registered under Composition Scheme" [(ngModel)]="customTemplate.sections['header'].data['gstComposition'].label" (ngModelChange)='onFieldChange(null, null, null)'>
```

Comment:
Consider using normal `maxlength` attribute instead of `[maxlength]` property to avoid change detection for fixed value

Reply:
Done